package.json

```diff
@@ -36,8 +36,10 @@
     "typescript": "^2.1.5"
   },
   "dependencies": {
+    "@types/react-bootstrap": "^0.0.45",
     "asap": "https://github.com/alloy/asap.git",
     "react": "^15.4.2",
+    "react-bootstrap": "^0.30.7",
```

Comment:
This is a massive framework ( https://react-bootstrap.github.io/components.html ) was there discussion about whether this is worth it anywhere I can read?